src/software/new_geom/util/physics.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "software/new_geom/point.h"
+
+/**
+ * Calculates the future position of a moving object
+ *
+ * @pre The units of the parameters are consistent, e.g. accleration is in (units of
+ * initial_position)/(units of time_in_future)^2
+ *
+ * @param initial_position The initial position of the object
+ * @param initial_velocity The initial velocity of the object in position
+ * @param acceleration The constant acceleration applied to the object over time
+ * @param time_in_future The time in the future to estimate position
+ *
+ * @return future position of the object
+ */
+Point calculateFuturePosition(const Point &initial_position,
+                              const Vector &intial_velocity, const Vector &acceleration,
+                              double time_in_future);
```

Comment:
Physics isn't really a logical subset of geometry. I think this should probably be under `src/software` (not  _huge_ fan of adding more stuff at that level, but I think it's most logical place), or alternatively, if this is only used in the linear ball model, we could just put it there (though it would be trickier to test).

Reply:
Yeah I thought about it. Since it uses geom types, I thought it belonged there

Reply:
By that argument pretty much everything in the repo should be in the geom folder! :joy: - That said, I definitely see what you mean. I still think it doesn't belong under the geometry folder.

Reply:
Hmm, I don't see any reason why something like `physics ` should be in the `util` folder but other stuff like `geom` has it's own top level folder?

Reply:
Yea fair enough, top-level `physics` seems most reasonable

Reply:
moved